packages/apollo-federation/src/composition/__tests__/utils.test.ts

```diff
@@ -37,6 +41,8 @@ describe('Composition utility functions', () => {
         type Mutation {
           updateProduct: Product
         }
+
+        extend interface Account @key(fields: "id")
```

Comment:
@lennyburdette this interface needs the `id` field defined so the query planner can ask for the `{ id }` on that interface